### DISCUSSION TITLE
How should I cite Echopype?

### DISCUSSION THREAD

**Discussion by IanTBlack** [OSOceanAcoustics/echopype#924] [General]
I would appreciate an example of how the maintainers/contributors would prefer the software to be cited in literature.
For a URL, should I point toward GitHub or readthedocs.io?

**Comment by leewujung** [comment]
Thanks for asking! We have this citation link that is on the right of the repo page, right above the Release (looks like below). It goes to an arXiv preprint.
For a URL, I think pointing to the readthedocs page would be great.

<img width="246" alt="Screen Shot 2022-12-27 at 4 40 51 PM" src="https://user-images.githubusercontent.com/15334215/209740308-dea9a636-b9b1-459d-b13d-872144f5806e.png">

**Reply by IanTBlack** [reply]
Thank you for the pointer toward the right information!

**Comment by IanTBlack** [comment]
I never would have imagined that I would experience spam on GitHub. I am both appalled and intrigued.

The answer to my original question was obviously thought of in advance when the repo was created, hence the link made available on the main repo page. I just didn't look hard enough. I would say that this Discussion topic can be closed if  the ability to do so exists.

**Reply by gwleuverink** [reply]
This is a first.. I kind of appreciate the ingenuity though

**Reply by haztakki** [reply]
It's really odd. I have never seen this before. I've also never heard or participated in this repository, but I was still tagged. I've reported it to Github.

**Reply by IanTBlack** [reply]
I've reported the user to GitHub as well. 

@leewujung I think as a maintainer you have more flexibility on how to report this if this happens again in the future.

**Reply by leewujung** [reply]
Oh wow, I've never seen this before either. I'll report this if it happens again!

**Reply by leewujung** [reply]
Also, there doesn't seem to be an option to close this, but I think it's fine to just keep it here since the location of the citation link is not obvious.
